### PR TITLE
Add Flake Finder Fridays assets

### PR DESCRIPTION
**Pull request by hasheddan**
**Which issue(s) this PR fixes**:

n/a

Bootstraps new subdir for the Flake Finder Fridays livestream.

/assign @justaugustus @saschagrunert 
/cc @RobertKielty

### CODE REVIEW

File: contributors/devel/sig-release/flake-finders/episodes/000/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Flake Finder Fridays #0 
+
+February 5th 2021 ([Recording](https://youtu.be/Hqlm2h2AEvA))
+
+## Introduction
+
+This is the first episode of Flake Finder Fridays with Dan Mangum and Rob
```
Comment (justaugustus) [inline contributors/devel/sig-release/flake-finders/episodes/000/README.md:7]: Maybe include your GitHub usernames here?

**Review by justaugustus — COMMENTED** [review]
This is awesome, Dan!
I've left one comment, but feel free to lift the hold when ready.

/lgtm
/approve
/hold

File: contributors/devel/sig-release/flake-finders/episodes/000/README.md (hunk 1 above)
Comment (hasheddan) [reply contributors/devel/sig-release/flake-finders/episodes/000/README.md:10]: @justaugustus good idea :) updated :+1: could you re-add your LGTM here?

**Pull request merged**